### DISCUSSION TITLE
Shift + Mouse (text selection in viewer)

### DISCUSSION THREAD

**Discussion by ghost** [doublecmd/doublecmd#1084] [Suggestions]
### Description

It's very hard to work with big text files on viewer, since viewer doesn't have the select option via SHIFT + MOUSE.
As go to the point of selection start, press and keep on pressing SHIFT then scroll down till where you want the end of selection to be made and click MOUSE.

In current form of viewer it can only select via scroll, and scrolling a big portion of text is just hard.

https://user-images.githubusercontent.com/122004809/220616114-af522a9d-7732-4293-b151-fdc618eadaf8.mp4

### Proposed technical implementation details (optional)

Implementation of SHIFT + MOUSE, text selection will be very pro !
Thanks

**Comment by thrive4** [comment]
+1 would also add shift + arrow keys to the mix it is not possible,
at least with windows to expand the selection using the arrow keys,
normally left or right) while holding the shift key.
